lib/trento_web/controllers/session_controller.ex

```diff
@@ -21,26 +21,30 @@ defmodule TrentoWeb.SessionController do
     security: [],
     request_body:
       {"User login credentials", "application/json",
-       %Schema{
+       %OpenApiSpex.Schema{
          title: "LoginCredentials",
          type: :object,
+         additionalProperties: false,
          example: %{
            username: "admin",
            password: "thepassword"
          },
          properties: %{
-           username: %Schema{
+           username: %OpenApiSpex.Schema{
+             type: :string
+           },
+           password: %OpenApiSpex.Schema{
              type: :string
            },
-           password: %Schema{
+           totp_code: %OpenApiSpex.Schema{
```

Comment:
Could we add nullable:false here? If we do I think you should also omit the field in request instead of passing a possible null

Reply:
mmm, are you sure? This value shouldn't be `null` in any case.
But we don't pass `null`, simply we don't send the field. The payload field has a conditional addition.
Here the "wrong" thing is that `username` and `password` should be required fields, and the other not, but if I add them as required, the non backward compatible CI change starts complaining...
And as we are in a feature branch, it would complain in all the PRs.
Better to do it once the `user_management` branch is merged to `main`, once for all

Reply:
PD: The default value is `false` indeed: https://spec.openapis.org/oas/v3.0.3#schemaNullable

Reply:
you're right! It's totallly fine!